### PR TITLE
feat(css): Allow at-rule descriptors also contain mdn_url key

### PR DESCRIPTION
**Pull request by skyclouds2001**

### CODE REVIEW

File: css/at-rules.json [hunk 1]
```diff
@@ -123,7 +133,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@document"
   },
   "@font-palette-values": {
-    "syntax": "@font-palette-values <dashed-ident> { <declaration-list> }",
+    "syntax": "@font-palette-values <dashed-ident> {\n  <declaration-list>\n}",
```
Comment (bsmth) [inline css/at-rules.json:136]: Is this intentional?

**Review by bsmth — COMMENTED** [review]
One comment open about syntax changes, otherwise looking great, thanks.

File: css/at-rules.json (hunk 1 above)
Comment (skyclouds2001) [reply css/at-rules.json:136]: Yes, just a formatting adjustment to keep it consistent with other at-rules

**Review by bsmth — APPROVED** [review]
LGTM, thanks!

**Pull request merged**